packages/api/controllers/hackathon_register_controller.go

```diff
@@ -152,3 +161,49 @@ func FormatTeamName(teamName string) string {
 func CompareTeamNames(passedTeamName string, storedTeamName string) bool {
 	return FormatTeamName(passedTeamName) == FormatTeamName(storedTeamName)
 }
+
+func SendEmailToNewParticipant(fullName string, email string) {
+
+	requestURL := os.Getenv("MAILING_URL")
+
+	type Mailer struct {
+		Receiver string `json:"receiver"`
+		Html     string `json:"html"`
+		Subject  string `json:"subject"`
+	}
+
+	var reqBody Mailer
+
+	reqBody.Html = "<h1>HELLO</h1>"
+	reqBody.Receiver = email
+	reqBody.Subject = "TEST"
```

Comment:
Subject should probably be `Welcome to HackAUBG 5.0` or something like that.

Reply:
Please don't cut my legs